### DISCUSSION TITLE
Visualizations Appearing In Colab but not VSCode

### DISCUSSION THREAD

**Discussion by kentstephen** [developmentseed/lonboard#511] [General]
Hello,

I have been able to run lonboard in VSCode Jupyter using the Overture Maps notebook example. But when I try and use my own visualizations they do not render. However, when I run the same code in Colab, the maps appear. I'd rather run this locally. What should I look for? I've tried searching the discussions here and did not see anything. Hopefully its a simple fix.

**Comment by kylebarron** [comment]
1: Make sure you're on lonboard 0.9.1, not 0.9.0.

Then
- I'd try fully reloading vscode, if you haven't.
- If you run `import lonboard; lonboard.viz()`, does that show a map?
- Can you look if there are any errors in the console (option - command - I) on a mac

**Comment by kentstephen** [comment]
![Screenshot (327)](https://github.com/developmentseed/lonboard/assets/43362477/dd0e3160-3278-4515-bb0d-2132c268ac31)

I don't see any errors.

**Reply by kylebarron** [reply]
Browser console, not terminal. That's why I mentioned the keyboard shortcut

<img width="1103" alt="image" src="https://github.com/developmentseed/lonboard/assets/15164633/1931a04f-ac5d-4c86-94b4-3cef3582e04d">

**Reply by kentstephen** [reply]
![image](https://github.com/developmentseed/lonboard/assets/43362477/aeaafae3-155a-4a3c-938c-0c43f380db10)

I learned a new thing today. Thank you. I already updated lonboard so it's running.

**Reply by kentstephen** [reply]
is this what you meant?
![image](https://github.com/developmentseed/lonboard/assets/43362477/f2f05b36-6a15-48b7-9c92-86809e5a3684)

**Reply by kentstephen** [reply]
i reinstalled lonboard==0.9.0 to recreate and it didnt render, and i grabbed the console here

![image](https://github.com/developmentseed/lonboard/assets/43362477/9debb250-8116-4a30-b914-1993166bfb61)

**Reply by kylebarron** [reply]
Yeah, that was solved by https://github.com/developmentseed/lonboard/pull/508, but now you know how to check for JavaScript runtime errors in vscode 🙂

**Comment by kentstephen** [comment]
Ok I resolved, I was running the previous version. Thank you for your help.
